src/app/docs/model.js

```diff
@@ -36,8 +36,8 @@ angular
         $scope.parentsLength = Object.keys($scope.parents).length;
         var default_compiled = '\n-- compiled SQL not found for this model\n';
```

Comment:
We should update all these comments too.

Reply:
yes we should :)

Reply:
...and syntax highlighting, potentially

Reply:
Updated this but couldn't find where to touch to update syntax highlighting